### PR TITLE
JAV-543 Fix dns resolve

### PR DESCRIPTION
**Pull request by wujimin**
it's netty's bug.
latest vertx had upgrade netty dependence.
we can upgrade to latest vertx 3.5.0

### CODE REVIEW

File: foundations/foundation-vertx/src/main/java/io/vertx/ext/web/impl/MimeTypesUtils.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
+package io.vertx.ext.web.impl;
```
Comment (WillemJiang) [inline foundations/foundation-vertx/src/main/java/io/vertx/ext/web/impl/MimeTypesUtils.java]: Please add license header.

File: foundations/foundation-vertx/src/test/java/io/vertx/ext/web/impl/TestMimeTypesUtils.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
+package io.vertx.ext.web.impl;
```
Comment (WillemJiang) [inline foundations/foundation-vertx/src/test/java/io/vertx/ext/web/impl/TestMimeTypesUtils.java]: License header.

File: java-chassis-dependencies/pom.xml [hunk 3]
```diff
@@ -1,1001 +1,1001 @@
...
+    <artifactId>java-chassis</artifactId>
+    <version>0.4.1-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>java-chassis-dependencies</artifactId>
+  <packaging>pom</packaging>
+
+  <properties>
+    <jackson.version>2.8.10</jackson.version>
+    <vertx.version>3.5.0</vertx.version>
```
Comment (WillemJiang) [inline java-chassis-dependencies/pom.xml]: This file is reformated, Can we just update the version ?

File: foundations/foundation-vertx/src/main/java/io/vertx/ext/web/impl/MimeTypesUtils.java (hunk 1 above)
Comment (wujimin) [reply foundations/foundation-vertx/src/main/java/io/vertx/ext/web/impl/MimeTypesUtils.java]: done

File: foundations/foundation-vertx/src/test/java/io/vertx/ext/web/impl/TestMimeTypesUtils.java (hunk 2 above)
Comment (wujimin) [reply foundations/foundation-vertx/src/test/java/io/vertx/ext/web/impl/TestMimeTypesUtils.java]: done

File: java-chassis-dependencies/pom.xml (hunk 3 above)
Comment (wujimin) [reply java-chassis-dependencies/pom.xml]: first commit in yellow cloud, forgot if reformat it.
it's not important, right?

**Pull request merged**